test/e2e/tests/custom-rpc-history.spec.js

```diff
@@ -0,0 +1,155 @@
+const { strict: assert } = require('assert');
+const { regularDelayMs, largeDelayMs, withFixtures } = require('../helpers');
+
+describe('Stores custom RPC history', function () {
+  const ganacheOptions = {
+    accounts: [
+      {
+        secretKey:
+          '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+        balance: 25000000000000000000,
+      },
+    ],
+  };
+  it(`creates first custom RPC entry`, async function () {
+    await withFixtures(
+      {
+        fixtures: 'imported-account',
+        ganacheOptions,
+        title: this.test.title,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+
+        const rpcUrl = 'http://127.0.0.1:8545/1';
+        const chainId = '0x539'; // Ganache default, decimal 1337
+
+        await driver.clickElement('.network-display');
+        await driver.delay(regularDelayMs);
```

Comment:
[Comment:] Maybe we can remove these delay calls, as they are added long back when the selenium web driver was directly used. Since then we have switched to our own driver abstraction, and they were more robust and worked fine without the delays for me while migrating the other test specs.